src/reducers/contactList.jsx

```diff
@@ -1,56 +1,54 @@
-import {FILTER, DELETE} from '../actions';
+import { FILTER, DELETE } from '../actions';
 
 const initialState = {
   data: [
     {
       id: 1,
-      image: "http://via.placeholder.com/150x150/",
+      image: 'http://via.placeholder.com/150x150/',
       name: 'Matvei',
-      number: + 375446669966
+      number: '+375446669966',
     }, {
       id: 2,
-      image: "http://via.placeholder.com/150x150/",
+      image: 'http://via.placeholder.com/150x150/',
       name: 'Maria',
-      number: + 375449996699
+      number: '+375449996699',
     }, {
       id: 3,
-      image: "http://via.placeholder.com/150x150/",
+      image: 'http://via.placeholder.com/150x150/',
       name: 'Valera',
-      number: + 375443336669
+      number: '+375443336669',
     }, {
       id: 4,
-      image: "http://via.placeholder.com/150x150/",
+      image: 'http://via.placeholder.com/150x150/',
       name: 'FurryGod',
-      number: + 375299090900
+      number: '+375299090900',
     }, {
       id: 5,
-      image: "http://via.placeholder.com/150x150/",
+      image: 'http://via.placeholder.com/150x150/',
       name: 'Siel Phantomhive',
-      number: + 375266666900
-    }
+      number: '+375266666900',
+    },
   ],
   filteredData: [],
```

Comment:
A couple of words about store organization. Actually we don't need field 'filteredData', in addition to this field name 'data' is not descriptive, all that we need is contactList(array) and for example visibilityFilter(string). According to field visibilityFilter we can make decision which contacts need to be shown at the moment (we filter it in our component).
So, if you organize store as I said, you no longer need doFilter action, only setVisibilityFilter, and contacts will be filtered in component itself according to the visibilityFilter value.